src/main/java/teammates/ui/controller/AdminSearchPageAction.java

```diff
@@ -10,6 +10,7 @@
 import teammates.common.datatransfer.InstructorAttributes;
```

Comment:
Can we avoid using AdminSearchPageAction for this? Resetting Google ID is not related to search, although it appears in the search result page. Define a new action?